Alignment/OfflineValidation/test/eopTreeWriter_cfg.py

```diff
@@ -23,7 +23,7 @@
 
 # load configuration files
 process.load("Configuration.StandardSequences.MagneticField_cff")
-process.load("Configuration.Geometry.GeometryDB_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
@ASvyatkovskiy - yes, using DB or RecoDB (for GTs which do not need/have Sim geometry) would be a right thing to do. Thanks.